src/manifest.rs

```diff
@@ -16,8 +16,8 @@ pub const MANIFEST_FILE: &str = "Proto.toml";
 /// empty fields.
 #[derive(Debug, Clone, Serialize, Deserialize, PartialEq, Eq)]
 pub struct RawManifest {
```

Comment:
can we make this pub(crate)? not done by you initially but just recognizing it should be

Reply:
That makes sense, but I think it's even better to implement a `Manifest::from_string()` function so we don't need to expose this struct outside its module (similar to how I implemented `RawLockfile`).

Reply:
Made the raw struct private (and implemented a few to/from string traits to support converting the manifest to/from TOML).